docs/static/getting-started-with-logstash.asciidoc

```diff
@@ -144,6 +144,12 @@ WARNING: The repositories do not work with older rpm based distributions
 
 See the <<running-logstash,Running Logstash>> document for managing Logstash as a system service.
```

Comment:
To be consistent with how we introduce xrefs, I would say:  `"See <<running-logstash,Running Logstash>> for details about managing Logstash as a system service."` The link formatting makes it pretty clear that you're jumping to another page.

Now I have to admit that we're not always consistent. :-)